blog/_posts/2024-05-30-introductory-post.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Simplify Your Azure Kubernetes Service Connection Configuration with Service Connector"
+description: "Service Connector introduction"
+date: 2024-05-18
+author: Coco Wang
+categories: security # general, operations, networking, security, developer topics, add-ons
+---
+Workloads deployed on an Azure Kubernetes Service (AKS) cluster often need to access Azure backing resources, such as Azure Key Vault, databases, or AI services like Azure OpenAI Service. Users are required to manually configure [Microsoft Entra Workload ID](https://learn.microsoft.com/en-us/entra/workload-id/workload-identities-overview) or Managed Identities so their AKS workloads can securely access these protected resources.
+The [Service Connector](https://learn.microsoft.com/en-us/azure/service-connector/overview) integration greatly simplifies the connection configuration experience for AKS workloads and Azure backing services. Service Connector takes care of authentication and network configurations securely and follows Azure best practices, so you can focus on your application code without worrying about your infrastructure connectivity.
+ 
+![image](https://github.com/CocoWang-wql/AKS/assets/45681473/8c146088-0bd9-4095-a0c2-c802a135b4c2)
+Service Connector Action Breakdown     
+
+Before, in order to [connect from AKS pods to a private Azure backing services](https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster) using workload identity, users needed to perform the following actions manually:
+1.	Create a managed identity
+2.	Retrieve the OIDC issuer URL
+3.	Create Kubernetes service account
+4.	Establish federated identity credential trust
+5.	Grant permissions to access Azure Services
+6.	Deploy the application
+
+Now, Service Connector performs steps 2 to 5 automatically. Additionally, for Azure services without public access, Service Connector creates private connection components such as private link, private endpoint, DNS record, etc.   
+You can create a connection in the Service Connection blade within AKS.
```

Comment:
Can you create the connection any other way besides the portal? CLI, Bicep, TF, etc.